src/test/GradleTestDevModeActionsWSProjectNameTasks.ts

```diff
@@ -0,0 +1,265 @@
+import { expect } from 'chai';
+import { InputBox, Workbench,SideBarView, ViewItem, ViewSection,EditorView,DefaultTreeItem, DebugView } from 'vscode-extension-tester';
+import * as utils from './utils/testUtils';
+import * as constants from './definitions/constants';
+import path = require('path');
+
+describe('Devmode action tests for Gradle Project- Project name with space', () => {
+    let sidebar: SideBarView;
+    let debugView: DebugView;
+    let section: ViewSection;
+    let item: DefaultTreeItem;    
+    let tabs: string[];
+
+    before(() => {
+        sidebar = new SideBarView();
+        debugView = new DebugView();      
+    });
+
+it('getViewControl works with the correct label',  async() => { 
+   
+   const contentPart = sidebar.getContent();
+   section = await contentPart.getSection('Liberty Dashboard');   
+   console.log("Found Liberty Dashboard....");
+   expect(section).not.undefined; 
+ 
+}).timeout(25000);
+
+
+it('Open dasboard shows items - Gradle', async () => {
+
+      
```

Comment:
Please format the file and remove unwanted spaces.

Reply:
Formatted file and removed unwanted spaces. Thanks